src/main/resources/static/views/user/UserPage.html

```diff
@@ -0,0 +1,373 @@
+                                    <div class="card">
+                                        <a href="#"><img ng-src="https://goo.gl/bLja0p"
+                                                         class="card-img-top img-responsive"
+                                                         alt=""></a>
+                                        <div class="card-block">
+                                            <a href="#"><h4 class="card-title">The NOMATIC Backpack and Travel Pack</h4>
+                                            </a>
+                                            <div class="card-text">
+                                                <div>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+                                                    eiusmod
+                                                    tempor
+                                                    incididunt ut labore et
+                                                    dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
+                                                    exercitation
+                                                    ullamco
+                                                    laboris nisi ut aliquip
+                                                    ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
+                                                    voluptate
+                                                    velit
+                                                    esse cillum dolore eu
+                                                    fugiat nulla pariatur.
+                                                </div>
+                                            </div>
+                                        </div>
+                                        <div class="card-footer">
+                                            <div class="row">
+                                                <div class="col-xs-6">
+                                                    <div>min</div>
+                                                    <div><span class="fa fa-dollar"></span>100000</div>
+                                                </div>
+                                                <div class="col-xs-6">
+                                                    <div class="right-box">Still needed</div>
+                                                    <div class="right-box"><span class="fa fa-dollar"></span>9999</div>
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                                <div class="container col-md-4 col-xs-12">
+                                    <div class="card">
+                                        <a href="#"><img ng-src="https://goo.gl/e7PZpL"
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add an "alt" attribute to this image. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AImgWithoutAltCheck)